R/generate_qc_report.R

```diff
@@ -23,6 +27,8 @@ generate_qc_report <- function(library_id,
                                unfiltered_sce,
                                filtered_sce = NULL,
                                processed_sce = NULL,
+                               rmd_file = NULL,
+                               extra_params = c(),
```

Comment:
Since you use an `is.null` test later, we should set the default to `NULL`
```suggestion
                               extra_params = NULL,
```